File: 01realTimeComputeAddressOrderAmount/src/main/java/com/hhh/temp/tools/RedisTools.java
package com.hhh.temp.tools;

import redis.clients.jedis.Jedis;

/**
 * Created by huanghh on 2017/5/22.
 */
public class RedisTools {

    private static Jedis jedis = new Jedis("192.168.101.121");

    public static Jedis getJedis() {
        return jedis;
    }
}

File: 00tools/src/main/scala/com/hhh/temp/tools/RedisTools.scala
package com.hhh.temp.tools

import redis.clients.jedis.Jedis

/**
  * Created by huanghh on 2017/5/22.
  */
object RedisTools {
  private val jedis: Jedis = new Jedis("192.168.101.121");
  def getJedis(): Jedis = {
    jedis
  }
}
